### ISSUE TITLE
Implement Theme Switching / Light Mode

### ISSUE THREAD

**Issue by shawnborton** [Expensify/App#12260]
Note: This issue is part of the greater **[Implement Our New Branding in NewDot](https://github.com/Expensify/Expensify/issues/239140)** project. 

![image](https://user-images.githubusercontent.com/2319350/198701221-dcd8f39b-19a5-45f3-b883-5864c57e1971.png)

**What is it:**
Add the ability to switch themes in the app between light and dark modes. 

**Does it require a mini design doc:**
Yes, this will have lots of code changes

**Design finished:**
Yes

cc @GabiHExpensify

**Comment by quinthar** [comment]
Do we need to actually implement both light/dark mode?  What if to start we *only* implemented "dark mode", and just saved any kind of "mode switcher" for the future?  /cc @JmillsExpensify

**Comment by Luke9389** [comment]
I think we should implement both. 

For one, I think it'd be jarring to be locked into dark mode. Some folks won't want the dark theme. 

Also, one doesn't really block the other. We can work in parallel on the app makeover and implement a theme switcher at the same time (and often in the same PR). Overall I think it'll be much faster _not_ to separate these initiatives.

**Comment by shawnborton** [comment]
Adding a note that as part of [this conversation here](https://expensify.slack.com/archives/C01GTK53T8Q/p1667277018509469), we've decided to remove this one from the WAQ Project. That doesn't mean we shouldn't do this, but let's prioritize all other WAQ issues first before moving onto this one.

**Comment by parasharrajat** [comment]
These designs are really pro work. Loved the typography. Awesome work @shawnborton.

**Comment by grgia** [comment]
Relevant thread: https://expensify.slack.com/archives/C01GTK53T8Q/p1670282228733019
TLDR: Let's perfect Dark Mode first, though many people would appreciate having light mode back

**Comment by Luke9389** [comment]
Thanks for updating!

**Comment by Luke9389** [comment]
I'm unclear if this is still on hold. I think we can re-open a convo about this now that the WAQ initiative is no longer blocking stuff. What are your thoughts on moving forward with this one @shawnborton?

**Comment by shawnborton** [comment]
I think we can start to work on this one again, starting with some pre-design discussions based on the best way to implement theme switching. I think @grgia volunteered to lead that up!

**Comment by shawnborton** [comment]
Not overdue. I think this one could be a good option for an expert contributor though!

**Comment by JmillsExpensify** [comment]
I agree that this would be good for an expert contributor. What's holding us back at the moment?

**Comment by shawnborton** [comment]
I'm not sure what would be holding us back. We'd probably want to find an engineer on our side to buddy up with the expert contributor for this? I'm happy to help with all of the mocks, etc as well. 

Do you have a particular contributor/group in mind? 

Also cc @grgia as this is something you've worked on in the past.

**Comment by JmillsExpensify** [comment]
Nice, yeah seems like we just need someone to help take it on and work with the appropriate contributor. Let's bring to #bug-zero and align on the appropriate group to help.

**Comment by grgia** [comment]
I'm happy to continue with this! I got wrapped up with the footer project but I'm out of the woods with that. If now's a good time to restart it, we can rework the P/S statement we were working on to include working with expert contributors and I could post this in whatsnext or help with the discussion

**Comment by shawnborton** [comment]
Sweet! I think it makes sense for the Sign Up/In page to be treated as a priority above this one though, but if you have the capacity for both, that's awesome.

**Comment by JmillsExpensify** [comment]
Agree with @shawnborton. Let's get the new homepage wrapped up, this is really cool just not as important as our first experience.

**Comment by grgia** [comment]
Not overdue, in progress

**Comment by grgia** [comment]
Closing this in favor of the main tracking issue for now.
